domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/EntityToResponseDTOMapper.java

```diff
@@ -251,12 +251,14 @@
     @Mapping(source = "workBasketCaseFieldEntity.caseField.reference", target = "caseFieldId")
     @Mapping(source = "workBasketCaseFieldEntity.caseFieldElementPath", target = "caseFieldElementPath")
     @Mapping(source = "workBasketCaseFieldEntity.accessProfile.reference", target = "role")
+    @Mapping(source = "workBasketCaseFieldEntity.caseType.reference", target = "caseTypeId")
     @Mapping(expression = "java(workBasketCaseFieldEntity.getCaseField().isMetadataField())", target = "metadata")
     WorkBasketResultField map(WorkBasketCaseFieldEntity workBasketCaseFieldEntity);
 
     @Mapping(source = "searchCasesResultFieldEntity.caseField.reference", target = "caseFieldId")
     @Mapping(source = "searchCasesResultFieldEntity.caseFieldElementPath", target = "caseFieldElementPath")
     @Mapping(source = "searchCasesResultFieldEntity.accessProfile.reference", target = "role")
+    @Mapping(source = "searchCasesResultFieldEntity.caseType.reference", target = "caseTypeId")
```

Comment:
Do we need it for `SearchResultsField` too?

Reply:
ya.. just added that tooo